packages/react/src/__tests__/ReactProfilerComponent-test.internal.js

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
Split this set of tests out into separate module because mixing ReactDOM and ReactNoop within the same test module can cause failures that are tough to debug.